index.js

```diff
@@ -127,9 +158,39 @@ Use the game function below to do the following:
   HINT: While you can complete this with only conditionals based on strings, it may help to equate choice to a number when using Math.random()
 */
 
-function game(/*add your code here*/){
-    /*add your code here*/
+function game(userChoice){
+  function compChoice(){
+    let compHand=Math.random();
+    if (compHand>=0&&compHand<=.33){
+      return "scissors";
+    } else if (compHand>=.34&&compHand<=.66){
+      return "rock";
+    } else if (compHand>=.67&&compHand<=1){
+      return "paper";
+    }
+    }
+    let handOne=compChoice();
+    if (userChoice==="scissors"&&handOne==="scissors"){
+      return "it's a tie";
+    } else if (userChoice==="scissors"&&handOne==="paper") {
+      return "you win!";
+    } else if (userChoice==="scissors"&&handOne==="rock") {
+      return "you lose!";
+    } else if (userChoice==="rock"&&handOne==="scissors") {
+      return "you win!";
+    } else if (userChoice==="rock"&&handOne==="paper") {
+      return "you lose!";
+    } else if (userChoice==="rock"&&handOne==="rock") {
+      return "it's a tie";
+    } else if (userChoice==="paper"&&handOne==="scissors") {
+      return "you lose!";
+    } else if (userChoice==="paper"&&handOne==="paper") {
+      return "it's a tie";
+    } else if (userChoice==="paper"&&handOne==="rock") {
+      return "you win!";
+    }
 }
```

Comment:
Nice Job using` ===`  You should definitely use `===` when you can over `==`. 
This means you specify both type and value.